### PR TITLE
Primary key, foreign, comment

### PR DESCRIPTION
**Pull request by bayuadhi2121**

### CODE REVIEW

File: app/Http/Controllers/PersonController.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
+    {
+
+
+        $validatedData = $request->validate([
+            'nama' => 'required|max:255',
+            'kontak' => 'required',
```
Comment (audiadyan) [inline app/Http/Controllers/PersonController.php]: Kontak opsional

File: app/Http/Controllers/PersonController.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            'username' => 'required',
+            'status' => 'required'
+        ]);
+        Person::create($validatedData);
```
Comment (audiadyan) [inline app/Http/Controllers/PersonController.php:41]: Kurang password

File: database/migrations/2023_08_28_120404_create_persons_table.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('persons', function (Blueprint $table) {
+            $table->String('id_person', 20)->primary();
+            $table->String('nama', 50);
+            $table->String('username')->unique();
+            $table->String('password')->default('123456789');
```
Comment (audiadyan) [inline database/migrations/2023_08_28_120404_create_persons_table.php]: Gak di hash dulu ni?

**Review by audiadyan — APPROVED** [review]

File: app/Http/Controllers/PersonController.php (hunk 2 above)
Comment (bayuadhi2121) [reply app/Http/Controllers/PersonController.php:41]: udah deafult kan ?

**Review by bayuadhi2121 — COMMENTED** [review]
cek lagi migrasi person sama controller

File: app/Http/Controllers/PersonController.php (hunk 2 above)
Comment (audiadyan) [reply app/Http/Controllers/PersonController.php:41]: Ooo.. di migrasi tu

**Pull request merged**